### PR TITLE
ENH: add option for black or white ruler

### PR DESCRIPTION
**Pull request by pieper**
Works for both slice and threeD views.

Saves and restores with mrml scene.

Is propagated with other attributes when views are linked.

### CODE REVIEW

File: Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx [hunk 1]
```diff
@@ -457,6 +462,39 @@ int vtkMRMLAbstractViewNode::GetRulerTypeFromString(const char* name)
...
+      // invalid id
+      return "";
+    }
+}
+
+//-----------------------------------------------------------
+int vtkMRMLAbstractViewNode::GetRulerColorFromString(const char* name)
+{
+  if (name == nullptr)
+  {
```
Comment (lassoan) [inline Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx]: Indentation

Comment (pieper) [reply Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx]: I agree it's a bit odd, but this matches the style for GetRulerTypeAsString above so I figured it's better to stick with precedent.

Comment (pieper) [reply Libs/MRML/Core/vtkMRMLAbstractViewNode.cxx]: Ah, I see,  you were talking about GetRulerColorFromString - yes, that's an indent issue I copied from GetRulerTypeFrom string - I'll fix them both.

**Comment by pieper** [comment]
Yellow makes sense - I'll push an update in a minute.

**Comment by jcfr** [comment]
Nitpick: would be great if could squash the commit, GitHub now keep track of it and it make it easy to review the  history in the future and understand the changes as it was accepted in.

**Comment by pieper** [comment]
Yellow option added, squashed, and committed as r28283

**Comment by pieper** [comment]
Thanks @jcfr - I force-pushed the squashed commit here for future reference.

**Comment by lassoan** [comment]
Awesome, thank you!

**Pull request closed without merging**